core/src/main/scala/kafka/coordinator/transaction/TransactionCoordinator.scala

```diff
@@ -169,6 +169,7 @@ class TransactionCoordinator(brokerId: Int,
                   s"${Topic.TRANSACTION_STATE_TOPIC_NAME}-${txnManager.partitionFor(transactionalId)}")
                 responseCallback(initTransactionMetadata(newMetadata))
               } else {
+                info(s"Returning $error error code to client for $transactionalId's init pid request")
```

Comment:
The convention has been to write the transactionalId as `Returning $error error code to client for transactionalId $transactionalId in InitPidRequest`. 

The reason is that sometimes we may want to grep out all messages relating to all transactionalId s, in which case we can just grep for 'transactionalId'.